src/archiefvernietigingscomponent/destruction/utils.py

```diff
@@ -0,0 +1,118 @@
+from datetime import date, datetime
+
+from django.shortcuts import render
+from django.utils.translation import gettext as _
+
+from zds_client import ClientError
+
+from archiefvernietigingscomponent.destruction.constants import (
+    ListItemStatus,
+    ReviewStatus,
+)
+from archiefvernietigingscomponent.destruction.models import (
+    DestructionList,
+    DestructionListReview,
+)
+from archiefvernietigingscomponent.destruction.service import (
+    fetch_process_type,
+    fetch_zaaktype,
+)
+
+
+class NoClientException(Exception):
+    """No client could be built"""
+
+    pass
+
+
+def get_looptijd(zaak: dict) -> int:
+
+    if zaak["einddatum"] != "":
+        end_date = datetime.strptime(zaak["einddatum"], "%Y-%m-%d")
+    else:
+        end_date = date.today()
+
+    return (end_date - datetime.strptime(zaak["startdatum"], "%Y-%m-%d")).days
+
+
+def get_vernietigings_categorie_selectielijst(zaaktype: str) -> str:
+    try:
+        zaaktype = fetch_zaaktype(zaaktype)
+    except ClientError as exc:
+        # If the zaaktype couldn't be retrieved, return an empty value
+        return ""
+
+    if not zaaktype.get("selectielijstProcestype"):
+        return ""
+
+    try:
+        process_type = fetch_process_type(zaaktype["selectielijstProcestype"])
+    except ClientError as exc:
+        # If the process type couldn't be retrieved, return an empty value
+        return ""
+
+    return str(process_type["nummer"])
+
+
+def get_destruction_list_archivaris_comments(destruction_list: DestructionList) -> str:
+    review = (
+        DestructionListReview.objects.filter(
+            destruction_list=destruction_list,
+            status=ReviewStatus.approved,
+            author__role__can_review_destruction=True,
+            author__role__can_view_case_details=False,
+        )
+        .order_by("created")
+        .last()
+    )
+
+    if not review:
+        return ""
+
+    return review.text
+
+
+def get_process_owner_comments(destruction_list: DestructionList) -> str:
+    review = (
+        DestructionListReview.objects.filter(
+            destruction_list=destruction_list,
+            status=ReviewStatus.approved,
```

Comment:
the same question as above - is `status=approved` required?

Reply:
When the reviewers advise for changes and the record manager makes the changes, the review process restarts from the beginning. So I thought that in the report it wouldn't be so useful to see all the comments that lead to changes (since in the report only the final version can be seen). 
But maybe we should add all the comments. What do you think?

Reply:
Now I understand, I agree with you, but we can make sure and ask product owners

Reply:
Clarified in meeting: the final comments are enough for now